src/fmeval/transforms/summarization_accuracy_metrics.py

```diff
@@ -0,0 +1,221 @@
+import ray
+import nltk
+import evaluate as hf_evaluate
+
+from typing import List, Dict, Any, Union
+from ray import ObjectRef
+from nltk import word_tokenize
+from nltk.translate import meteor_score
+
+from fmeval.util import require
+from fmeval.transforms.transform import Transform
+from fmeval.transforms.util import validate_call
+from fmeval.helper_models import BertscoreModel
+
+
+METEOR_SCORE = "meteor"
+ROUGE_SCORE = "rouge"
+BERT_SCORE = "bertscore"
+
+# rouge constants
+ROUGE_1 = "rouge1"
+ROUGE_2 = "rouge2"
+ROUGE_L = "rougeL"
+
+ROUGE_TYPES = [ROUGE_1, ROUGE_2, ROUGE_L]
+
+
+def _load_meteor_modules() -> None:  # pragma: no cover
+    """Load helper modules required by meteor metric.
+
+    :returns: None
+    """
+    nltk.download("wordnet")
+    nltk.download("punkt")
+    nltk.download("omw-1.4")
+
+
+class MeteorScore(Transform):
+    """This Transform augments an input record with the METEOR metric, computed from target and model outputs.
+
+    METEOR is a metric for text similarity between the machine-produced summary
+    and human-produced reference summaries.
+    Unigrams can be matched based on their surface forms, stemmed forms,
+    and meanings; furthermore, METEOR can be easily extended to include more
+    advanced matching strategies. Once all generalized unigram matches
+    between the two strings have been found, METEOR computes a score for
+    this matching using a combination of unigram-precision, unigram-recall, and
+    a measure of fragmentation that is designed to directly capture how
+    well-ordered the matched words in the machine translation are in relation
+    to the reference.
+    """
+
+    def __init__(
+        self,
+        input_keys: List[str],
+        output_keys: List[str],
+        target_output_key: str,
+        model_output_key: str,
+        load_meteor_modules: bool = True,
+    ):
+        """MeteorScore initializer.
+
+        :param input_keys: A two-element list containing the keys corresponding to the target output and LLM output.
+        :param output_keys: A single-element list containing the output key for this Transform,
+            which corresponds to the METEOR metric that gets computed.
+        :param target_output_key: The key corresponding to the target output.
+        :param model_output_key: The key corresponding to the model (LLM) output.
+        :param load_meteor_modules: Whether to load the meteor helper modules. This needs to be performed only once.
+        """
+        require(
+            set(input_keys) == {target_output_key, model_output_key},
+            f"input_keys to MeteorScore should be {target_output_key, model_output_key}.",
+        )
```

Comment:
Question: Why are `target_output_key` and `model_output_key` being passed as additional parameters, if `input_keys` should already contain these? 

Is there a way we can remove the need to pass in repeated parameters?

Reply:
Yeah, so the reason I added them as additional parameters is that we need to be able to distinguish between the two. We could rely on a convention where `input_keys[0]` is always interpreted to be the target output key for example, but we both know that relying on conventions that aren't explicitly enforced by the code is a recipe for disaster.

Right now, the `Transform` ABC implementation coupled with the way we create copies of transforms in `execute` pretty much forces all transforms to have `input_keys` and `output_keys` as their first two arguments. So even though in this case, we'd ideally just not pass `input_keys`, under the current structure, we have to.

I'd like to make a follow up PR where I change things so that this is no longer the case, as I agree that it results in ugly, redundant code.